charts/merlin/templates/_config.tpl

```diff
@@ -0,0 +1,61 @@
+{{- define "merlin.contextRef" -}}
+{{- $ := index . 0 }}
+{{- $reference := index . 1 }}
+{{ $.Values.imageBuilder.contextRef | default (printf "refs/tags/%s" $reference) }}
+{{- end -}}
+
+{{- define "merlin.renderedConfig" -}}
+{{- $ := index . 0 }}
+{{- $rendered := index . 2}}
+{{- if $rendered }}
+# this is because merlin artifact versions have no v prefix
+# NOTE: Remove the substr once merlin artifacts are released with v prefix
```

Comment:
Wow thanks for catching this 😅 🙏

Reply:
haha turing, xp and mlp don't have this issue, only merlin atm, but it's not a blocker